reflex-todomvc/default.nix

```diff
@@ -1 +1,17 @@
-import ((import <nixpkgs> {}).fetchFromGitHub (builtins.fromJSON (builtins.readFile ./github.json)))
+{ mkDerivation, reflex, reflex-dom, file-embed, cabal-macosx
+}:
+
+mkDerivation {
+  pname = "reflex-todomvc";
+  version = "0.1";
+  src = builtins.filterSource (path: type: baseNameOf path != ".git") ./.;
+  isExecutable = true;
+  isLibrary = true;
+  buildDepends = [
+    reflex
+    reflex-dom
+    file-embed
+    cabal-macosx
+  ];
+  license = null;
+}
```

Comment:
This file shouldn't be changed; instead, a PR should be made to reflex-todomvc, and then the github.json in this directory should be updated to point to the new revision.